[iOS][SDK] updated DEVELOPMENT.md (#117)
NimbleEdge/deliteAI: sdks/ios/docs/DEVELOPMENT.md

# DeliteAI iOS SDK

To build the **DeliteAI iOS SDK** locally and run tests, follow these steps:

## Running the SDK Locally Through Example App

To effectively test and observe the functionalities of the DeliteAI iOS SDK locally, you can utilize the provided `NimbleNetExample` application. Follow these detailed steps to get it up and running:

1.  **Install Third-Party Binaries:**

>     From the root directory of your cloned DeliteAI iOS SDK repository, execute the following command to install necessary third-party binaries:

Reviewer: ```suggestion
  Run the following command from the **ROOT of the repo** to download all the required dependencies:
```

# DeliteAI iOS SDK

To build the **DeliteAI iOS SDK** locally and run tests, follow these steps:

## Running the SDK Locally Through Example App

To effectively test and observe the functionalities of the DeliteAI iOS SDK locally, you can utilize the provided `NimbleNetExample` application. Follow these detailed steps to get it up and running:

1.  **Install Third-Party Binaries:**

>     Run the following command from the **ROOT of the repo** to download all the required dependencies:

Reviewer: ```suggestion
    Run the following command to download all the required dependencies:
```

    ```bash

>     $GIT_ROOT/setup.sh --sdk ios

Reviewer: ```suggestion
      cd $(git rev-parse --show-toplevel) && ./setup.sh --sdk ios
```

    ```

1.  **Build NimbleNet.xcframework:**
    This command requires **CMake** to be installed on your machine. If you don't have CMake, you can install it using Homebrew:

    ```bash
    brew install cmake
    ```

    Build the `NimbleNet.xcframework` by running the following command from the project's root directory:

    ```bash

>     $GIT_ROOT/sdk/ios/script/build-deliteAI-static.sh

Reviewer: ```suggestion
    cd $(git rev-parse --show-toplevel) && ./sdk/ios/script/build-deliteAI-static.sh
```
